AEPCore/Sources/eventhub/AEPError.swift

```diff
@@ -20,5 +20,9 @@ import Foundation
     case callbackTimeout = 1
     case callbackNil = 2
     case none = 3
+    case serverError = 4
```

Comment:
Are these an artifact from some other work?  I don't see them used anywhere in this PR.